app/src/main/java/com/github/palFinderTeam/palfinder/map/MapsFragment.kt

```diff
@@ -282,10 +289,20 @@ class MapsFragment : Fragment(), OnMapReadyCallback, GoogleMap.OnMarkerClickList
             val position = LatLng(meetUp.location.latitude, meetUp.location.longitude)
             val options = MarkerOptions().position(position).title(meetUp.uuid)
 
-            if (meetUp.markerId != null && iconPack != null) {
-                val icon = iconPack!!.getIcon(meetUp.markerId)
-                val bitmap = icon?.drawable?.toBitmap(64, 64)
-                if (bitmap != null) options.icon(BitmapDescriptorFactory.fromBitmap(bitmap))
+            val markerId = meetUp.markerId ?: 2
```

Comment:
Also here the 2, maybe add a constant named DEFAULT_MARKER_ID

Reply:
It's fixed and I wille test it hext sprint